libs/sdk-core/src/fiat.rs

```diff
@@ -64,8 +64,7 @@ fn convert_to_fiat_currency_with_id(id: String, info: CurrencyInfo) -> FiatCurre
 
 #[tonic::async_trait]
 impl FiatAPI for BreezServer {
-    /// Retrieve all available fiat currencies defined in a local configuration file
-    fn list_fiat_currencies(&self) -> Result<Vec<FiatCurrency>> {
+    fn list_fiat_currencies_unfiltered(&self) -> Result<Vec<FiatCurrency>> {
```

Comment:
I don't think we should rename it. From the point of view of this trait there is no filtering.

Reply:
I only renamed it to differentiate from the other method, which would be identically named but have a different effect.

Should I then rename the `BreezServices` method? (and all callers)

Or leave both names identical, but risk confusion when someone reads the docs?

Reply:
Good point, will update the PR.

Reply:
Done.

Ready for review again @roeierez @erdemyerebasmaz